sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusManagementAsyncClient.java

```diff
@@ -342,48 +792,363 @@
     }
 
     /**
-     * Creates a queue with its context.
+     * Updates a subscription with the given {@link SubscriptionDescription}. The {@link SubscriptionDescription} must
+     * be fully populated as all of the properties are replaced. If a property is not set the service default value is
+     * used.
```

Comment:
Is there a online documentation which we can point user to for list of properties and default values ? More methods below have same comment.

Reply:
The likelihood that users click on the documentation first before trying this out and having it fail is low.
There is a `@see` annotation to each of these methods if they want more complete information.